classes/opponentPhaseOrbit.lua

```diff
@@ -0,0 +1,60 @@
+-- The template opponent phase
+
+local opponentPhaseOrbit = {}
+opponentPhaseOrbit.__index = opponentPhaseOrbit
+
+
+local ORBIT_START_ANG = -math.pi / 2
+local ORBIT_ANG_SPEED = 1.2 	-- How many radians per second should the opponent advance in orbit?
+local ORBIT_RADIUS_X = 400
+local ORBIT_RADIUS_Y = 250
+
+local ORBIT_FOCAL_X = ARENA_WIDTH / 2
+local ORBIT_FOCAL_Y = ARENA_HEIGHT / 2
+
+
+opponentPhaseOrbit.SPAWN_X = ORBIT_FOCAL_X + ORBIT_RADIUS_X * math.cos(ORBIT_START_ANG)
+opponentPhaseOrbit.SPAWN_Y = ORBIT_FOCAL_Y + ORBIT_RADIUS_Y * math.sin(ORBIT_START_ANG)
```

Comment:
These are here as optional variables you can assign to a "class" metatable. `SPAWN_X` and `SPAWN_Y` will inform the OpponentSpawner where to draw the circle